COLLABORATOR_GUIDE.md

```diff
@@ -471,9 +471,9 @@ $ git node land $PRID
 ```
 
 If it's the first time you ever use `node-core-utils`, you will be prompted
-to type the password of your GitHub account in the console so the tool can
-create the GitHub access token for you. If you do not want to do that, follow
-[the guide of `node-core-utils`][node-core-utils-credentials]
+to type the password of your GitHub account and the 2FA code in the console
```

Comment:
The onboarding doc writes out `two-factor authentication` and links to the GitHub help page.